support/http/logging_middleware_test.go

```diff
@@ -11,6 +11,21 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+// set "X-Forwarded-For" header to test LoggingMiddlewareWithOptions
+func setXFFMiddleWare(next stdhttp.Handler) stdhttp.Handler {
+	return stdhttp.HandlerFunc(func(w stdhttp.ResponseWriter, r *stdhttp.Request) {
+		r.Header.Set("X-Forwarded-For", "203.0.113.195")
+		next.ServeHTTP(w, r)
+	})
+}
+
+// set "Content-MD5" header to test LoggingMiddlewareWithOptions
+func setCMDFiveMiddleWare(next stdhttp.Handler) stdhttp.Handler {
```

Comment:
Better to write MD5 rather than MDFive as it connects better with its meaning and how people are used to read it.

Also, abbreviating `Content` to `C` is a big leap. Better not to spare words and make it clearer.

```suggestion
// setContentMD5Middleware sets "Content-MD5" header to test LoggingMiddlewareWithOptions
func setContentMD5Middleware(next stdhttp.Handler) stdhttp.Handler {
```

Reply:
Addressed here a18f383